### ISSUE TITLE
build hiredis failed with Python3.5.0

### ISSUE THREAD

**Issue by ipfans** [redis/hiredis-py#35]
Build hiredis with Python3.5.0, it will be error: "AssertionError"

More details as below:

```
....
  gcc -pthread -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Ivendor -I/usr/local/python3/include/python3.5m -c src/hiredis.c -o build/temp.linux-x86_64-3.5/src/hiredis.o
  gcc -pthread -shared build/temp.linux-x86_64-3.5/src/reader.o build/temp.linux-x86_64-3.5/src/hiredis.o -Lbuild/temp.linux-x86_64-3.5 -lhiredis_for_hiredis_py -o build/lib.linux-x86_64-3.5/hiredis/hiredis.cpython-35m-x86_64-linux-gnu.so
  installing to build/bdist.linux-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.linux-x86_64
  creating build/bdist.linux-x86_64/wheel
  creating build/bdist.linux-x86_64/wheel/hiredis
  copying build/lib.linux-x86_64-3.5/hiredis/hiredis.cpython-35m-x86_64-linux-gnu.so -> build/bdist.linux-x86_64/wheel/hiredis
  copying build/lib.linux-x86_64-3.5/hiredis/__init__.py -> build/bdist.linux-x86_64/wheel/hiredis
  copying build/lib.linux-x86_64-3.5/hiredis/version.py -> build/bdist.linux-x86_64/wheel/hiredis
  running install_egg_info
  running egg_info
  writing dependency_links to hiredis.egg-info/dependency_links.txt
  writing hiredis.egg-info/PKG-INFO
  writing top-level names to hiredis.egg-info/top_level.txt
  warning: manifest_maker: standard file '-c' not found

  reading manifest file 'hiredis.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no previously-included files found matching 'vendor/hiredis/example*'
  warning: no previously-included files found matching 'vendor/hiredis/text*'
  writing manifest file 'hiredis.egg-info/SOURCES.txt'
  Copying hiredis.egg-info to build/bdist.linux-x86_64/wheel/hiredis-0.2.0-py3.5.egg-info
  running install_scripts
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-build-z8jh6ss0/hiredis/setup.py", line 81, in <module>
      'Topic :: Software Development',
    File "/usr/local/python3/lib/python3.5/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/usr/local/python3/lib/python3.5/distutils/dist.py", line 955, in run_commands
      self.run_command(cmd)
    File "/usr/local/python3/lib/python3.5/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/mnt/env3/lib/python3.5/site-packages/wheel/bdist_wheel.py", line 213, in run
      archive_basename = self.get_archive_basename()
    File "/mnt/env3/lib/python3.5/site-packages/wheel/bdist_wheel.py", line 161, in get_archive_basename
      impl_tag, abi_tag, plat_tag = self.get_tag()
    File "/mnt/env3/lib/python3.5/site-packages/wheel/bdist_wheel.py", line 155, in get_tag
      assert tag == supported_tags[0]
  AssertionError

  ----------------------------------------
  Failed building wheel for hiredis
Failed to build hiredis
```

**Comment by badboy** [comment]
Currently I can't reproduce that. What did you do to cause this?

**Comment by graphaelli** [comment]
I actually see the same error for any `bdist_wheel` on my machine - using a virtualenv with python3.5 (as above) but on mac osx (unlike linux as above) but haven't tracked it down yet.

**Comment by jhedev** [comment]
I was able to reproduce it by:

```
pip3.5 install wheel==0.24
python3.5 setup.py wheel
```
Looks like this is a bug that was fixed in version `0.25`:

https://bitbucket.org/pypa/wheel/issues/146/wheel-building-fails-on-cpython-350b3

After upgrading wheel everything works fine for me.

**Comment by ipfans** [comment]
I install success after upgrade wheel to 0.26. And it confirm to be a bug in wheel package.
